Bd okta 702470 fix typos (#4687)
okta/okta-developer-docs: packages/@okta/vuepress-site/books/api-security/sanitizing/attack-vectors/index.md

Inputs are everywhere, often only evident in hindsight. User input and file uploads are just the tip of the iceberg, but what if we consider more than input and instead the code itself? Here are a couple of examples to illustrate this point.

### Your Dependencies

> Do you trust all of your dependencies? How about all of the transitive dependencies of your application? It's common practice for an application to list the versions of its dependencies and licenses (the latter might even be required depending on the license). The popular Node package manager (npm) has had a few projects that have contained [maliciously formed license fields](https://blog.npmjs.org/post/80277229932/newly-paranoid-maintainers). In another npm incident, [packages ran malicious scripts](https://iamakulov.com/notes/npm-malicious-packages/) after installation that uploaded the user's environment variables to a third party.

Reviewer: ```suggestion
Do you trust all of your dependencies? How about all of the transitive dependencies of your application? It's common practice for an application to list the versions of its dependencies and licenses (the latter might even be required depending on the license). The popular Node package manager (npm) has had some projects that contained [maliciously formed license fields](https://blog.npmjs.org/post/80277229932/newly-paranoid-maintainers). In another npm incident, [packages ran malicious scripts](https://iamakulov.com/notes/npm-malicious-packages/) after installation that uploaded the user's environment variables to a third party.
```
